src/main/java/com/mraof/minestuck/world/lands/terrain/FloraLandType.java

```diff
@@ -38,6 +34,8 @@
 	private static final Vec3d fogColor = new Vec3d(0.5D, 0.6D, 0.9D);
 	private static final Vec3d skyColor = new Vec3d(0.6D, 0.8D, 0.6D);
 	
+	private static final BlockState BLOOD = MSBlocks.BLOOD.getDefaultState();
```

Comment:
I'd advice against static block variables like this. Generally it should be avoided as if the initialization is triggered before blocks (or whatever else) has been created, the block will be null, causing the thing to fail. While I don't think it's likely to happen here, I'd still suggest sticking to that principle.

Reply:
Changed!